plugins/cuda/cuda_plugin.c

```diff
@@ -151,27 +156,35 @@ static int launch_cuda_checkpoint(const char **args, char *buf, int buf_size)
 			return -1;
 		}
 
+		if (WEXITSTATUS(status) != EXIT_SUCCESS)
```

Comment:
It can be killed by signal and we probably want to see its output in this case too. Actually, I think we can always print it to the log.

Reply:
I have updated the patch to also print the output when the child process is killed by signal.

This function is also used to get the output from `cuda-checkpoint --help`. To prevent cluttering the logs, we skip printing this output if `exit_code == EXIT_SUCCESS`.
```
$ cuda-checkpoint --help
CUDA checkpoint and restore utility.
Version 555.42.06. Copyright (C) 2024 NVIDIA Corporation. All rights reserved.

Operations:
--get-state --pid <pid>
        Prints the current checkpoint state of the process specified by <pid>

--action lock | checkpoint | restore | unlock --pid <pid> [--timeout <ms>]
        Performs the specified action on <pid>.
        For the lock action a timeout can be provided, the lock operation will wait up to <ms> milliseconds for the operation to succeed.

--toggle --pid <pid>
        Toggles the CUDA state in the specified process between the running and checkpointed states

--get-restore-tid --pid <pid>
        Retrieves the CUDA restore thread ID of the process specified by <pid>

Options:
--pid|-p <pid>
        The pid upon which to perform the operation

--timeout|-t <timeout>
        Optional timeout that can be specified for the lock action in milliseconds

--help|-h
        Print this help message
```